rootfs/scheduler/__init__.py

```diff
@@ -1024,9 +1019,9 @@ def _create_rc(self, namespace, name, image, command, **kwargs):  # noqa
                 secret = self._get_secret('deis', 'objectstorage-keyfile').json()
                 self._create_secret(namespace, 'objectstorage-keyfile', secret['data'])
 
-            l["image"] = image
+            l["slug_url"] = image
```

Comment:
That should happen in `Release.image` and not in the scheduler. I can look at doing that, right now I made it so all but External Registries get the Internal Registry prepended. Just to confirm, only build packs should have that excluded, right?